experiments/metavariable-analysis/entropy/string-literal-entropy.yaml

```diff
@@ -0,0 +1,37 @@
+rules:
+  - id: string-literal-entropy
+    patterns:
+      - pattern: "..."
+      - metavariable-analysis:
+          analyzer: entropy
+          metavariable: "..."
+    message: Found a high-entropy string.
+    languages:
+      - bash
```

Comment:
@clintgibler @minusworld @lejliu @inkz @colleend this is a (slightly) altered list from 
`semgrep-core --help | grep "bash" | sort | uniq | sed "s/, /\r\n- /g" | pbcopy`

Please help sanity check for correctness in case I missed something or deleted the wrong version of a language (there are shortname dupes for many of these)